samples/keypoint_detection/module.yml

```
@@ -0,0 +1,52 @@
+name: ${oc.env:MODULE_NAME, 'keypoint_detection'}
```

```suggestion
name: keypoint_detection
```

For consistency with incoming changes for Denis' PR.